libs/sdk-backend-spi/src/workspace/userGroups/index.ts

```diff
@@ -0,0 +1,69 @@
+// (C) 2019-2021 GoodData Corporation
+import { ObjRef } from "@gooddata/sdk-model";
+import { IPagedResource } from "../../common/paging";
+
+/**
+ * User Group
+ * @alpha
+ */
+export interface IWorkspaceUserGroup {
+    /**
+     * Stored user group reference
+     */
+    ref: ObjRef;
+    /**
+     * Stored user group id (unfortunately some of bear API need group URI and some its id, so we need to keep both of them)
```

Comment:
We should not mention bear explicitly here.

Reply:
done